src/pyjion/pyjit.h

```diff
@@ -83,7 +83,7 @@ static PY_UINT64_T HOT_CODE = 0;
 static PY_UINT64_T jitPassCounter = 0;
 static PY_UINT64_T jitFailCounter = 0;
 
-static PyjionSettings g_pyjionSettings;
+extern PyjionSettings g_pyjionSettings;
```

Comment:
My understanding is that each TU that included `pyjit.h` had its own `g_pyjionSettings`. If we mark it as `extern` and then declare it in `pyjit.cpp`, they should all be reading from and writing to the same memory.

Reply:
awesome! thanks. I wanted to update some of the unit tests to verify that some `assertOptimized` is the opposite when the level is 0